persistence/sql.go

```diff
@@ -0,0 +1,48 @@
+package persistence
+
+import (
+	"context"
+	"fmt"
+	"github.com/jackc/pgx/v4"
+	"log"
+	"math/rand"
+	"os"
+	"time"
+)
+
+const (
+	PostgresSchemaEnvVar = "POSTGRES_SCHEMA"
+
+	CreateSchemaIfNotExists = "CREATE SCHEMA IF NOT EXISTS"
+	SetSearchPathTo         = "SET search_path TO"
+	CreateTableIfNotExists  = "CREATE TABLE IF NOT EXISTS"
+	TableName               = "users"
+	TableSchema             = "(id int)"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+func connectAndInitializeDatabase(postgresUrl string) error {
+	ctx := context.TODO()
+	conn, err := pgx.Connect(context.Background(), postgresUrl)
+	if err != nil {
+		log.Fatalf("Unable to connect to database: %v\n", err)
+	}
+	schema := os.Getenv(PostgresSchemaEnvVar)
```

Comment:
I know we originally decided to put this as an env variable, but maybe it should be in the config?

Reply:
Could we tend to this minor infra/wiking piece, split it out into the original PR, and merge it independently of the SQL migrations?

Reply:
https://github.com/pokt-network/pocket/pull/75